config/reader.go

```diff
@@ -31,28 +30,54 @@ func AddConfigEnv(path string) (*ProductConfig, error) {
 	return product, nil
 }
 
-func loadConfigEnv(path string) (config *ProductConfig, err error) {
-	viper.AddConfigPath(path)
-	viper.SetConfigName("product")
-	viper.SetConfigType("yaml")
-	viper.AutomaticEnv()
-
-	err = viper.ReadInConfig()
-	if err != nil {
+func loadEnv(fullPath string) (config *ProductConfig, err error) {
+	if err = setEnv(fullPath); err != nil {
 		return nil, err
+	}
+
+	config = &ProductConfig{}
+	config.TFVars = os.Getenv("ENV_TFVARS")
+	config.Product = os.Getenv("ENV_PRODUCT")
 
+	if config.TFVars == "" || (config.TFVars != "k3s.tfvars" && config.TFVars != "rke2.tfvars") {
+		fmt.Printf("unknown tfvars: %s\n", config.TFVars)
```

Comment:
Do we want to use logger instead of fmt here?

Reply:
to use logger here we will need a refactor on the calls, which is not the case of this PR